src/java/test/org/apache/zookeeper/test/LoadFromLogTest.java

```diff
@@ -136,21 +113,10 @@ public void testLoad() throws Exception {
      */
     @Test
     public void testLoadFailure() throws Exception {
-        final String hostPort = HOST + PortAssignment.unique();
-        // setup a single server cluster
-        File tmpDir = ClientBase.createTmpDir();
-        ClientBase.setupTestEnv();
-        ZooKeeperServer zks = new ZooKeeperServer(tmpDir, tmpDir, 3000);
-        // So we have at least 4 logs
         SyncRequestProcessor.setSnapCount(50);
```

Comment:
iiuc the server has already been started in setUp method and as a result this will have no effect. Am I reading that right. Instead you might need to set this then call setUp.